hr_timesheet_sequence/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Timesheet Sequence",
+    "summary": """
+        Adds field sequence.
```

Comment:
Incomplete description, better this one:
```suggestion
        Adds sequence to timesheets within a day
```